src/Razor/test/VSCode.FunctionalTest/tests/CodeAction.test.ts

```diff
@@ -8,8 +8,10 @@ import { before, beforeEach } from 'mocha';
 import * as path from 'path';
 import * as vscode from 'vscode';
 import {
+    DoCodeAction,
+    GetCodeActions,
+    MakeEditAndFindDiagnostic,
```

Comment:
Remove the condition in [line 23](https://github.com/dotnet/aspnetcore-tooling/pull/1698/files#diff-54600b4e81ead513ec1edbb9d0e01db7R25)? Ditto for CodeActions2_2

Reply:
We can try, but like it says in the PR intro these still failed for me when I ran them locally, all this PR does is get them in a state where it would even be possible for them to pass.